assets/css/pages.css

```diff
@@ -1,7 +1,9 @@
 #profile_edit form, #post_edit form {
+  background-color: white;
   max-width: 70rem;
   margin: 0 auto 5rem auto;
   padding: 0 1rem;
+  padding: 4vw;
```

Comment:
@avogel3 there's a `padding` already, can we replace one by the other?

Reply:
how did I miss that 😅